### PR TITLE
docs: span component tag

### PR DESCRIPTION
**Pull request by ZStriker19**
## Description

We recommend integrations add `component:<integration_name>` tag to all spans.

## Checklist
- [x] Followed the [library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines) when writing a release note.
- [x] Add additional sections for `feat` and `fix` pull requests.
- [x] [Library documentation](https://github.com/DataDog/dd-trace-py/tree/1.x/docs) and/or [Datadog's documentation site](https://github.com/DataDog/documentation/) is updated. Link to doc PR in description.

## Reviewer Checklist
- [x] Title is accurate.
- [x] Description motivates each change.
- [x] No unnecessary changes were introduced in this PR.
- [x] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Tests provided or description of manual testing performed is included in the code or PR.
- [x] Release note has been added and follows the [library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines), or else `changelog/no-changelog` label added.
- [x] All relevant GitHub issues are correctly linked.
- [x] Backports are identified and tagged with Mergifyio.

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -248,6 +248,7 @@ All integrations live in ``ddtrace/contrib/`` and contain at least two files,
 
 It is preferred to keep as much code as possible in ``patch.py``.
 
+All spans generated by the integration must add the tag ``component:<integration_name>`` to each span.
```
Comment (mabdinur) [inline docs/contributing.rst:251]: Can we add an example (or link to one).

@wconti27 @brettlangdon Are there any other tags that should be set in every integration that we should document here? (ex: span.kind, _dd.measured, _dd.tracer_kr, etc.)

Comment (ZStriker19) [reply docs/contributing.rst:251]: sure, sounds good

Comment (brettlangdon) [reply docs/contributing.rst:251]: let's add them as we make them consistent, wdyt @wconti27? (asking that you could own updating this doc with any new PRs you have)

**Review by wconti27 — DISMISSED** [review]

**Review by mabdinur — APPROVED** [review]

File: docs/contributing.rst [hunk 2]
```diff
@@ -261,6 +261,8 @@ All integrations live in ``ddtrace/contrib/`` and contain at least two files,
...
+`Example of component tag being set in Flask integration <https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/contrib/flask/patch.py#L486-L487>`_.
```
Comment (emmettbutler) [inline docs/contributing.rst:265]: If the 1.x branch ever changes, this link could end up pointing to something other than the intended bit of code. You can turn it into a GitHub permalink by pressing `y` on that page.

**Review by emmettbutler — APPROVED** [review]

File: docs/contributing.rst (hunk 2 above)
Comment (Yun-Kim) [reply docs/contributing.rst:265]: Today I learned 😮

Comment (Yun-Kim) [inline docs/contributing.rst:265]:
```suggestion
`Example of component tag being set in Flask integration <https://github.com/DataDog/dd-trace-py/blob/a01c18f20de2348ed34bde3ac2fe7a1e010a2d38/ddtrace/contrib/flask/patch.py#L486-L487>`_.
```

**Review by Yun-Kim — CHANGES_REQUESTED** [review]

File: docs/contributing.rst (hunk 2 above)
Comment (ZStriker19) [reply docs/contributing.rst]: same, merged Yun's suggestion, should be good for final approvals

**Review by emmettbutler — APPROVED** [review]

**Review by mabdinur — APPROVED** [review]

**Review by Yun-Kim — APPROVED** [review]

**Review by majorgreys — APPROVED** [review]

**Pull request merged**